backend/shared/actions.ts

```diff
@@ -47,12 +46,12 @@ export const sendMessageToConnection = async (
 ): Promise<unknown> => {
   const { handler, connectionId } = config;
   try {
-    await handler
-      .postToConnection({
-        ConnectionId: connectionId,
-        Data: JSON.stringify(message),
-      })
-      .promise();
+    await handler.postToConnection({
+      ConnectionId: connectionId,
+      // Strings seem to work as well, which might indicate that the SDK types
+      // are too strict. Also, converting to Uint8Array is cumbersome.
+      Data: JSON.stringify(message) as unknown as Uint8Array,
```

Comment:
Ah yes of course, a Buffer is also a `UInt8Array` and can directly accept strings. Doing this, I now get a `Blob` in the frontend instead of a string. Which may be ok, I just need to convert it to text, but this is yet another reason to test this on development first.

Reply:
done